src/doc/rustc/src/platform-support/managarm.md

```diff
@@ -0,0 +1,63 @@
+# `*-unknown-managarm-mlibc`
+
+**Tier: 3**
+
+## Target Maintainers
+
+- [@no92](https://github.com/no92)
+- [@64](https://github.com/64)
+- [@Dennisbonke](https://github.com/Dennisbonke)
+
+## Requirements
+
+This target is cross-compiled. There is currently no support for `std` yet. It generates binaries in the ELF format.
+
+## Building the target
+
+For now, building a patched LLVM with [our patches located here](https://github.com/managarm/bootstrap-managarm/tree/master/patches/llvm) is necessary.
```

Comment:
I don't think any other targets currently need a patched LLVM to build (of course, newer LLVMs might work _better_). If I remember correctly, the loongarch targets waited until we had updated to a version of LLVM with sufficient support to link `libcore` before being merged. 

@no92 is there urgency to land this PR now or can we wait until an LLVM release picks up support for managarm?

Reply:
There's no urgency, so we're happy to merge our changes into LLVM first if that makes more sense.

But it would be nice if we could get this merged fairly soon after our changes hit LLVM master, because it blocks us upstreaming our changes to other Rust crates, and that's the bulk of the maintenance effort on our end: supporting locally patched crates (and sometimes multiple versions thereof!) has proved quite tricky to manage